File: 2000번대/2600~2699/2675.java
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		Scanner  sc = new Scanner(System.in);
		int count = sc.nextInt();
		for (int i = 0; i< count; i++) {
			int loop = sc.nextInt();
			String needChange = sc.next();
			for (int k =0; k < needChange.length(); k++) {
				for (int j =0; j < loop; j++) {
					System.out.print(needChange.charAt(k));
				}
			}
			System.out.println();
		}
	}
}

File: 2000번대/2600~2699/2675.kt
import java.util.*

fun main(args : Array<String>){
    val sc:Scanner = Scanner(System.`in`)
    val count = sc.nextInt()
    for (i in 1..count){
        var loop = sc.nextInt()
        var needChange = sc.next()
        for (k in 0..needChange.length-1){
            for (j in 0..loop-1){
                print(needChange[k])
            }
        }
        println();
    }
}
